cpp/src/parquet/arrow/arrow_schema_test.cc

```diff
@@ -889,6 +889,62 @@ Status ArrowSchemaToParquetMetadata(std::shared_ptr<::arrow::Schema>& arrow_sche
   return Status::OK();
 }
 
+TEST_F(TestConvertParquetSchema, ParquetVariant) {
```

Comment:
Is there a reason this test doesn't seem to exercise the Parquet Variant type, nor the Arrow VariantExtensionType? @wgtmac @neilechao

Reply:
I guess this test case was added by a commit before adding the extension type.